citadel/comparison.md

```diff
@@ -236,9 +242,16 @@ Ogre 1.x engine | ✓ | ✓
 Ogre 2.x engine | ✕ | ✓
 Optix engine | ✕ | ✓ Partial support
 Custom engine plugins | ✕ | [Issue](https://github.com/ignitionrobotics/ign-rendering/issues/100)
-Sky | ✓ | [Issue](https://github.com/ignitionrobotics/ign-rendering/issues/98)
+Sky | ✓ | ✕  (available from Edifice)
 Fog | ✓ |
 Material scripts | ✓ (Ogre 1.x scripts) | Does not apply
+Physically Based Rendering (PBR) | ✕ | ✓ (with engines that support it, like Ogre 2)
+Normal maps | ✓ | ✓
+Ambient occlusion maps | ✕  | ✓
```

Comment:
Oh I got it from [here](https://github.com/ignitionrobotics/ign-gazebo/blob/c9f815af2423462791c411f46b295d7bcb6bcf21/src/Conversions.cc#L297-L299). So that's just future-proofing?

Reply:
Removed: 2f9a8c8

Reply:
oh yeah it's one of the PBR map that's not yet supported by ogre2